### ISSUE TITLE
Bug when running mkdocs build

### ISSUE THREAD

**Issue by nathanaherne** [squidfunk/mkdocs-material#877]
## Description

I am not 100% certain if it is mkdocs or the material theme but I get the following error when building the container or running mkdocs build locally.

Ive not had issues with this container/local build for a over a year, it usually just builds without issue.

### Expected behavior

I expect it to build without issue.

### Actual behavior

I am getting the following error:

ERROR   -  Error building page 'index.md': object of type 'NoneType' has no len() 
Traceback (most recent call last):
  File "/usr/local/bin/mkdocs", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/mkdocs/__main__.py", line 163, in build_command
    ), dirty=not clean)
  File "/usr/local/lib/python2.7/site-packages/mkdocs/commands/build.py", line 295, in build
    _build_page(file.page, config, files, nav, env, dirty)
  File "/usr/local/lib/python2.7/site-packages/mkdocs/commands/build.py", line 216, in _build_page
    output = template.render(context)
  File "/usr/local/lib/python2.7/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/usr/local/lib/python2.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/nathanaherne/git/cruisin_co/www_cruisin_co/material/main.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/Users/nathanaherne/git/cruisin_co/www_cruisin_co/material/base.html", line 112, in top-level template code
    {% include "partials/tabs.html" %}
  File "/Users/nathanaherne/git/cruisin_co/www_cruisin_co/material/partials/tabs.html", line 9, in top-level template code
    {% include "partials/tabs-item.html" %}
  File "/Users/nathanaherne/git/cruisin_co/www_cruisin_co/material/partials/tabs-item.html", line 15, in top-level template code
    {% if (nav_item.children | first).children | length > 0 %}
TypeError: object of type 'NoneType' has no len()

### Steps to reproduce the bug

I have the latest version of mkdocs and the latest version of material. I am installing them via pip.

### Package versions

* Python: `python --version`
Python 2.7.14

* MkDocs: `mkdocs --version`
mkdocs, version 1.0.4 from /usr/local/lib/python2.7/site-packages/mkdocs (Python 2.7)

* Material: `pip show mkdocs-material | grep -E ^Version`
Version: 3.0.4

### Project configuration

``` yaml
The contents of your mkdocs.yml
```

### System information

* OS: [The operating system you're running]
I am using Alpine linux in the container and OSX locally. Both have the same issue.

* Browser: [The browser used, if relevant]

**Comment by squidfunk** [comment]
Closing due to inactivity. If you still experience the same problem, feel free to re-open.

**Comment by nathanaherne** [comment]
Hi, sorry about inactivity, I have been working on another project.

Yes, I override partials, as far as I remember, all I have done is remove the RHS TOC. I use the extra.css file

```
@media only screen and (min-width: 60em) {
  .md-content {
    margin-right: 0;
  }
}

.md-sidebar--secondary {
  display: none;
}
```

**Comment by nathanaherne** [comment]
I have not tried that but I am only using that for production. I am getting the build issue on my local OSX machine (without Docker).

**Comment by squidfunk** [comment]
I'm asking because if it works with the image, it's probably a problem with your setup. Could you try that?

**Comment by nathanaherne** [comment]
Ok, I understand now.

I tried to solve another way, I deleted the local copy of Material in my mkdocs site (which I pointed to using the mkdocs.yml config for a custom template). I hadn't changed the Material file structure, so there was no point using a custom folder. I just used the pip installed version of Material. It built fine on Docker Alpine.

In saying the above, I am having an issue with internal links but I do not think it is an issue with Material theme.
